webapp/components/abstract-test-file-results-table.js

```diff
@@ -78,10 +80,10 @@ class AbstractTestFileResultsTable extends WPTColors(TestRunsBase) {
           <td class$="[[ colorClass(result.status) ]]">
             <code>[[ subtestMessage(result) ]]</code>
             <template is="dom-if" if="[[result.screenshots]]">
-              <paper-button onclick="[[compareReferences(result)]]">
+              <a class="ref-button" href="[[ computeAnalyzerURL(result.screenshots) ]]">
                 <iron-icon icon="image:compare"></iron-icon>
-                Compare
-              </paper-button>
+                <span>COMPARE</span>
```

Comment:
Compare &rarr; COMPARE is because previously some style capitalized the text, right?

Reply:
Yes.